apps/figma-broker/transformers/typography.js

```diff
@@ -52,10 +50,7 @@ export const toTypography = (figmaNode, name) => {
     textDecoration,
     textTransform: textCase ? `${textCase}CASE` : null,
     fontStyle: italic ? 'italic' : null,
-  }
-
-  if (textAlign !== 'left') {
-    typography = R.set(textAlignLens, textAlign, typography)
+    textAlign,
```

Comment:
Add back the lowercasing? `R.toLower(textAlignHorizontal)` a lot of unecessary updates, and its all uppercase in tokens files now 🙈

Reply:
Yes, I was wondering about that 🤔 Text-transform is uppercase too…

Reply:
Hmm, yeah, it should be lowercase to... Don't you think?

Reply:
Yes, changed that too.